src/pybel/schema/edge.schema.json

```diff
@@ -0,0 +1,29 @@
+{
+    "$schema": "https://json-schema.org/draft/2019-09/schema#",
+    "title": "BEL Edge",
+    "description": "Top-level schema to validate JSON files containing an edge.",
+    "type": "object",
+    "properties": {
+        "subject": {"$ref": "modifier.schema.json"},
```

Comment:
Hmm I think i've made this pretty misleading, since it says "source" and "subject". I better update the SBEL export. Anyway, the idea is that the entity and all of the modifiers (that are edge specific) actually get taken apart and stored in different parts. Once you write a unit test for this, you'll see what the problem is. Try running

```python
import pybel
from pybel.examples.sialic_acid_graph
pybel.to_sbel_file(sialic_acid_graph, 'test.bel.jsonl')
```
to get some example data